ship-it-inc/ship-it
# #167446500 Implement login with google

#### What does this PR do?
It implements google login and jwt authentication
#### Description of Task to be completed?
- set up jwt token
- set up google authentication
#### How should this be manually tested?
- Fetch the branch `git fetch ft-user-google-auth-1674465007`
- Check out into the branch `ft-user-google-auth-167446500`
- visit localhost:3000/api/v1/auth/google
- You should be registered successfully and receive jwt token

#### Any background context you want to provide?
N/A
#### What are the relevant pivotal tracker stories?
[167446500](https://www.pivotaltracker.com/story/show/167446500)
